wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
```

Comment:
It's good practice to group imports as follows:

```console
standard library imports
<empty line>
third party imports
<empty line>
imporst from current package
```

The sort each individual block alphabetically by the package name.

In this case they are all packages from the standard library (AFAIK), so there shouldn't be any empty lines, and the imports should look like this:

```py
import logging
import requests
import socket
import urllib3
```